lib/galaxy/authnz/managers.py

```diff
@@ -35,7 +38,7 @@
     Strategy,
 )
 
-OIDC_BACKEND_SCHEMA = files("galaxy.authnz.xsd") / "oidc_backends_config.xsd"
+OIDC_BACKEND_SCHEMA = resource_path(__package__, "xsd/oidc_backends_config.xsd")
```

Comment:
Out of curiosity, why is this preferable over e.g. `files(__package__) / "xsd" / "oidc_backends_config.xsd"` ?

Reply:
They are equivalent, both are better than the previous version since `galaxy.authnz.xsd` is not really a package. `resource_path()` seems a bit more explicit and it's a couple less function calls, that's all.